cms/djangoapps/contentstore/views/certificate_manager.py

```diff
@@ -0,0 +1,309 @@
+"""
```

Comment:
Why did you move all of this into a new file? That seems unnecessary and not relevant to the view refactor?

Reply:
We were facing a circular import issue when importing `CertificateManager` from views into the serializer and then importing the serializer into the view. `CertificateManager` was defined in the view and used in many places, fixing the circular import issue by scoping the import was not a good solution, so we moved `CertificateManager` into a separate file and imported it into the view and serializer.